app/models/friendship.rb

```diff
@@ -0,0 +1,7 @@
+class Friendship < ApplicationRecord
+  validates :adder, presence: true
```

Comment:
It's redundant to validate adder and added. Your associations below already took care of that for you